src/Symfony/Component/Cache/Traits/RedisTrait.php

```diff
@@ -146,9 +147,13 @@ public static function createConnection($dsn, array $options = [])
             throw new InvalidArgumentException(sprintf('Invalid Redis DSN: %s', $dsn));
         }
 
+        if (isset($params['redis_sentinel']) && !class_exists(\Predis\Client::class)) {
+            throw new CacheException(sprintf('Redis Sentinel support requires the "predis/predis" package: %s', $dsn));
+        }
+
         $params += $query + $options + self::$defaultConnectionOptions;
 
-        if (null === $params['class'] && \extension_loaded('redis')) {
+        if (null === $params['class'] && !$params['redis_sentinel'] && \extension_loaded('redis')) {
```

Comment:
`!isset(...)`

Reply:
isset everywhere, or strict comparison with `null` everywhere btw

Reply:
Making the change but I'm not quite clear why this is different than redis_cluster which is doing simple truth checks in several places (L157, 252, 271).

Reply:
The difference is that "redis_cluster" is a boolean, while the new option is a string.

Reply:
ah, okay